eng/tools/oav-runner/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@azure-tools/oav-runner",
+  "private": true,
+  "type": "module",
+  "version": "1.0.0",
+  "main": "dist/src/main.js",
+  "bin": {
+    "oav-runner": "cmd/oav-runner.js"
+  },
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "test": "vitest --run --printConsoleTrace=true",
+    "pretty-check": "prettier \"**/*.ts\" --check",
+    "prettyfy": "prettier \"**/*.ts\" --write"
```

```suggestion
    "build": "tsc --build",
    "test": "vitest",
    "test:ci": "vitest run --coverage --reporter=verbose",
    "prettier": "prettier \"**/*.js\" --check",
    "prettier:write": "prettier \"**/*.js\" --write"
```

align with existing tools